src/main/java/ua/edu/internship/interview/service/dto/skill/SkillDto.java

```diff
@@ -0,0 +1,9 @@
+package ua.edu.internship.interview.service.dto.skill;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+
+@Data
+@EqualsAndHashCode(callSuper = true)
+public class SkillDto extends SkillBaseDto {
```

Comment:
Why do we need this SkillDto? I haven't found any usage of it

Reply:
This DTO is needed in next pull request. Discussed